xl2times/transforms.py

```diff
@@ -292,27 +319,32 @@ def merge_tables(
 
     for key, value in groupby(sorted(tables, key=lambda t: t.tag), lambda t: t.tag):
         group = list(value)
-        if not all(
-            set(t.dataframe.columns) == set(group[0].dataframe.columns) for t in group
-        ):
-            cols = [(",".join(g.dataframe.columns), g) for g in group]
-            logger.warning(
-                f"Cannot merge tables with tag {key} as their columns are not identical"
-            )
-            for c, table in cols:
-                logger.info(
-                    f"  {c} from {table.range}, {table.sheetname}, {table.filename}"
-                )
-        else:
-            df = pd.concat([table.dataframe for table in group], ignore_index=True)
-
-            match key:
-                case datatypes.Tag.fi_comm:
-                    model.commodities = df
-                case datatypes.Tag.fi_process:
-                    model.processes = df
-                case _:
-                    result[key] = df
+
+        if len(group) == 0:
+            continue
+
+        df = pd.concat([table.dataframe for table in group], ignore_index=True)
+        result[key] = df
```

Comment:
I had to roll this bugfix in here (with the performance changes) to make it work with Austimes.
There weren't any objections to this change in my initial PR discussion so hopefully this is ok?

This removes the constraint that the columns must be identical before merging, and replaces it with a warning if there are extra columns found after the merge.

Reply:
Sure, this shouldn't cause any issues. I am planning to expand the meta data in veda-tags.json to enable more harmonisation of the tables before merging.